HOWTO.rst

```diff
@@ -1632,6 +1632,24 @@ I/O type
 	space exceeds 2^32 blocks. If it does, then **tausworthe64** is
 	selected automatically.
 
+.. option:: sprandom=bool
+
+        SPRandom is a method designed to rapidly precondition large SSDs for
+        random write workloads. It divides the storage into equally sized regions
+        and calculates the required invalidation rate per region.
+        SPRandom requires **random_generator** to be set to **lfsr**.
+
```

Comment:
This should be written to communicate to a user who knows nothing about what this feature does.

How about this?

SPRandom is a method designed to rapidly precondition SSDs for
steady state random write workloads. It divides the device into equally sized regions
and writes the device's entire physical capacity one time selecting offsets so that the regions have a distribution of invalid blocks matching the distribution that occurs at steady state. Default: false.

It doesn't require `random_generator=lfsr` since this will be set for the user if it is unset.